lib/solargraph/rails/schema.rb

```diff
@@ -0,0 +1,100 @@
+module Solargraph
+  module Rails
+    class Schema
+      ColumnData = Struct.new(:type, :ast)
+
+      RUBY_TYPES = {
+        decimal: 'BigDecimal',
+        float: 'BigDecimal',
+        integer: 'Integer',
+        date: 'Date',
+        datetime: 'ActiveSupport::TimeWithZone',
+        string: 'String',
+        boolean: 'Boolean',
+        text: 'String',
+        jsonb: 'Hash',
+        json: 'Hash',
+        bigint: 'Integer',
+        uuid: 'String',
+        inet: 'IPAddr'
+      }
+
+      def self.instance
+        @instance ||= self.new
+      end
+
+      def self.reset
+        @instance = nil
+      end
+
+      def initialize
+        @schema_present = File.exist?('db/schema.rb')
```

Comment:
Would it work if we have `db/structure.sql` instead of `db/schema.rb` in our rails app?

Reply:
it won't, for that you'd have to parse SQL